release-tools/scripts/manifest.yml

```diff
@@ -28,7 +28,7 @@ urls:
     prod: s3://artifacts.opendistroforelasticsearch.amazon.com/
     #snapshots: s3://staging.artifacts.opendistroforelasticsearch.amazon.com/snapshots/
     releases: s3://staging.artifacts.opendistroforelasticsearch.amazon.com/releases/
-    releases_final_build: 
+    releases_final_build: 573103324 
```

Comment:
There is a space at the end. Remove it just incase it causes path issues

Reply:
Is it rc-number or just number?

Reply:
Just number.

Reply:
ok